### ISSUE TITLE
Issues training TacoTron

### ISSUE THREAD

**Issue by anthonycope** [NVIDIA/NeMo#731]
I'm running into some difficulty training the Tacotron model using NeMo. I'm attempting to train with the "Male US" dataset found here: https://www.caito.de/2019/01/the-m-ailabs-speech-dataset/

I'm currently training using my local desktop while we get our server configured.  In order to run on my 2080 Super, I've reduced the batch size to 16, and turned on mixed precision (amp_opt_level=O1). However, I have been unable to produce any intelligible results when training. My most recent attempt is currently at just over 40k steps, but the evaluation loss does not seem to be consistently going down, and the model is not reaching alignment. 

Is it possible that I simply need to run the training for longer? I was able to train the LJ-Speech model with similar settings and get acceptable results after about ~20k steps.

![image](https://user-images.githubusercontent.com/3255661/84687726-2d23a300-af0c-11ea-8a06-59cbd798ae39.png)

Any input would be appreciated. Thanks!

**Comment by blisc** [comment]
Please try the [tacotron2_preprocedefaults](https://github.com/NVIDIA/NeMo/tree/tacotron2_preprocedefaults) and use either the `tacotron2_dev_a.yaml` or the `tacotron2_dev_b.yaml` config files.

Please train tacotron2 in O0 mode.

Also, note that we have no 16kHz WaveGlow checkpoints and as a result we do not currently have WaveGlow checkpoints for the tacotron dev yaml files.

**Comment by anthonycope** [comment]
Thanks for the input. I've actually converted the audio to 22kHz in order to work with the existing WaveGlow configuration. Will changing the sample rate to 22 KHz in the config files in the mentioned branch work correctly?

**Comment by blisc** [comment]
Changing the config file will require you to mostly likely have to retrain Waveglow.
Note that regardless of the sample_rate, the dev configs in the tacotron2_preprocedefaults branch will not be fully compatible with the pre-trained waveglow checkpoints.

If you want to retain compatibility with the pre-trained waveglow checkpoints, then please use the main branch, and do the following:

- Use O0 optimization, that is to say use float32 training
- Set trim_silence to True inside the datalayer

Since you have resampled your audio, please ensure that you resampled with a higher quality filter

**Comment by anthonycope** [comment]
okay thanks! I'll try training with the parameters you've suggested and report back.
